backend/plugin/db/oracle/oracle.go

```diff
@@ -39,6 +42,7 @@ type Driver struct {
 	databaseName     string
 	serviceName      string
 	schemaTenantMode bool
+	dbVersion        int
```

Comment:
From life of a query perspective, we are still getting the version on every query since driver isn't reused.